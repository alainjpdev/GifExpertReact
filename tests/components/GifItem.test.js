import { screen, render } from "@testing-library/react"
import { GifItem } from "../../src/components/GifItem"




describe('Pruebas en <GifItem/>', () => { 
    
    const title= "DRAGON BALL"
    const url= "http://www.yomero.com"
    
    test('debe de hacer match con el snapshot', () => { 
        
        

        const { container } = render(<GifItem title={title} url={ url}/>)
        expect( container).toMatchSnapshot();

        // screen.debug()
     })

     test('debe de mostrar la imagen con el URL y el ALT', () => { 
     
       const { container } = render(<GifItem title={ title } url={ url }/>)

        const { src, alt } = screen.getAllByRole('img')
        expect( src).toBe( url ); 
      })
 })