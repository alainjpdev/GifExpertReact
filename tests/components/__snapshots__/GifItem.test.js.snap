// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifItem/> debe de hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="DRAGON BALL"
      src="http://www.yomero.com"
    />
    <p>
      DRAGON BALL
    </p>
  </div>
</div>
`;
